### DISCUSSION TITLE
Feature Request: add language modifier to Title Generation

### DISCUSSION THREAD

**Discussion by finnaGIT** [open-webui/open-webui#7787] [General]
# Feature Request

**Is your feature request related to a problem? Please describe.**
Titles may be generated in a user's non-native language depending on the model, rendering the titles useless in sorting through chat history. For example, I'm use qwen2.5 right now but many titles are generated by qwen in Chinese.

**Describe the solution you'd like**
Leverage the user's global WebUI Settings Language variable selected in **Settings >⚙️General** menu to supplement the default Title Generation prompt to respond in the user's chosen language. For example, the default prompt may be modified to read:
```
...
Respond only in {{language}}.
...
```

**Describe alternatives you've considered**
A modifier in the **Admin Panel > 🖥️ Interface** to input the instance's language in which responses should be generated. This solution would likely affect all users, so it is preferred to implement at a user level.

**Additional context**
-none-

**Comment by finnaGIT** [comment]
hi @tjbck , i see this was moved to discussions. meta: why are Feature Requests an option in the 'New Issues' area? just want to learn the proper channels to engage, so you can stay focused on the project and less on moderating :) thanks

**Comment by smonux** [comment]
You can configure your own prompt in Admin Settings -> Interface (my openwebui is in spanish, I guess that's the name)

![imagen](https://github.com/user-attachments/assets/be5f62f2-a41c-49cb-bd19-9d85ae0475e4)

The default is something like this:

Create a concise, 3-5 word title with an emoji as a title for the chat history, in the given language. Suitable Emojis for the summary can be used to enhance understanding but avoid quotation marks or special formatting. RESPOND ONLY WITH THE TITLE TEXT.\n\nExamples of titles:\n📉 Stock Market Trends\n🍪 Perfect Chocolate Chip Recipe\nEvolution of Music Streaming\nRemote Work Productivity Tips\nArtificial Intelligence in Healthcare\n🎮 Video Game Development Insights\n\n

It's strange that qwen is generating chinese titles. I use it and I haven't observed that.

**Comment by finnaGIT** [comment]
would be helpful to have the option to set language without modifying the entirety of the prompt. not sure why this was moved to discussions.
